custom_reports/object_management_api/definitions/custom_dimension_details.yaml

```diff
@@ -7,7 +7,7 @@ allOf:
         type: object
         required:
           - scope
-          - website_id
+          - slot
```

Comment:
Does `required` in openapi have different meaning? It appears in every case here, so it doesn't look like a mistake

Reply:
yes, I was looking at wrong repo.